### DISCUSSION TITLE
Is there a way to translate once strings from different versions of a software?

### DISCUSSION THREAD

**Discussion by tuxayo** [WeblateOrg/weblate#6737] [Q&A]
I don't know which are the best practices for managing translations for software having multiple versions supported.

I'm working on a project where we are considering moved to Weblate. And currently, when a new version N is released, the strings (.po) of the N-1 version are copied on the translation platform to a new project for the version N.
Then it's all independent and when a translation across all versions isn't complete, translators who are a bigshare using the N-2 or N-1 version of the software must take the time to copy the their work to the more recent versions. And if they don't, they will lose their work when upgrading to new versions.

Is there an organization made possible by Weblate that can avoid duplication of translation work when managing multiple versions of a software? Or maybe the possible ways to handle this don't impact the use of Weblate?

**Comment by nijel** [comment]
Use components for the versions and enable translation propagation. See https://docs.weblate.org/en/latest/faq.html#how-do-i-translate-several-branches-at-once
